src/webui/www/private/views/preferences.html

```diff
@@ -878,6 +878,14 @@
                     <input type="text" id="asyncIOThreads" style="width: 15em;" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="stopTrackerTimeout">QBT_TR(Stop Tracker Timeout:)QBT_TR[CONTEXT=OptionsDialog]&nbsp;</label>
```

Comment:
You should add a `?` libtorrrent documentation help link like in the other settings